scripts/install

```diff
@@ -76,7 +76,8 @@ ynh_install_app_dependencies $pkg_dependencies
 tmpdir="$(ynh_smart_mktemp)"
 pushd "$tmpdir"
 	wget https://golang.org/dl/go$GO_VERSION.linux-$architecture.tar.gz
-	rm -rf /usr/local/go && tar -C /usr/local -xzf go$GO_VERSION.linux-$architecture.tar.gz
+	ynh_secure_remove  /usr/local/go
```

Comment:
Eeeh that looks a bit savage but okay ? :thinking:

Reply:
there is an helper if you want to try => https://github.com/YunoHost-Apps/Experimental_helpers/blob/master/ynh_install_go/ynh_install_go

Reply:
(Apparently go is not needed anymore according to the new commits here, or am i missing something :thinking: )